app/models/application.rb

```diff
@@ -176,4 +181,17 @@ def funding_eligibility(with_funded_place:)
 
     eligible_for_funding && (funded_place.nil? || funded_place)
   end
+
+  def validate_funded_place?
+    accepted? && errors.blank? && cohort&.funding_cap?
+  end
+
+  def eligible_for_funded_place
```

Comment:
Is it worth having these shared validations in a concern so we're not duplicating them?

Reply:
There is slight differences to both methods, including how the errors are added. I'm not sure if its worth splitting it out.